skimage/util/dtype.py

```diff
@@ -244,6 +244,8 @@ def _convert(image, dtype, force_copy=False, uniform=False):
         if force_copy:
             image = image.copy()
         return image
+    else:
+        dtype = np.float64
```

Comment:
Did you mean `dtypeobj_out`? I think `dtype` is never used from this point onwards...

Reply:
Good catch, it should be `dtype_out` ;)